testers/testers/py/markus_python_tester.py

```diff
@@ -142,21 +142,10 @@ def _run_pytest_tests(self, test_file):
         with open(os.devnull, 'w') as null_out:
             try:
                 sys.stdout = null_out
-                legacy_venv_dir = self.specs.get('legacy_virtualenv')
                 verbosity=self.specs.get('pytest_tb_format', 'short')
-                if legacy_venv_dir is None:
-                    with tempfile.NamedTemporaryFile(mode="w+", dir=this_dir) as sf:
-                        pytest.main([test_file, '--junitxml', f'--tb={verbosity}', sf.name], plugins=[AddDocstringToJunitXMLPlugin()])
-                        results = list(self._parse_junitxml(sf))
-                else:
-                    try:
-                        with tempfile.NamedTemporaryFile(dir=this_dir, delete=False) as sf:
-                            tmp_file_path = sf.name
-                        pytest_exec = os.path.join(legacy_venv_dir, 'bin', 'pytest')
-                        subprocess.run([pytest_exec, '--junitxml', f'--tb={verbosity}', tmp_file_path, test_file])
-                        results = list(self._parse_junitxml(tmp_file_path))
-                    finally:
-                        os.unlink(tmp_file_path)
+                with tempfile.NamedTemporaryFile(mode="w+", dir=this_dir) as sf:
+                    pytest.main([test_file, '--junitxml', f'--tb={verbosity}', sf.name], plugins=[AddDocstringToJunitXMLPlugin()])
```

Comment:
This is probably the same thing you fixed in 1.7.1?

Reply:
yeah. I'm adding the fix for that in Part 7 since that will make other changes to the testers themselves